### ISSUE TITLE
drop_file_ops

### ISSUE THREAD

**Issue by karthik** [karthik/rdrop2#87]
copy, move, delete and create_folder

Relevant URLs

```
move_url <- "https://api.dropbox.com/1/fileops/copy"
move_url <- "https://api.dropbox.com/1/fileops/move"
create_url <- "https://api.dropbox.com/1/fileops/delete"
create_url <- "https://api.dropbox.com/1/fileops/create_folder"
```

Notes from @scheidec

__Copy__

- Code: https://github.com/karthik/rdrop2/blob/master/R/drop_file_ops.R
- Change from v1: /1/fileops/copy to v2 /2/files/copy
- Return values and metadata have changed
- V1 docs: https://www.dropbox.com/developers-v1/core/docs#fileops-copy
- V2 docs: https://www.dropbox.com/developers/documentation/http/documentation#files-copy
‘root’ not available in v2, from_path and to_path have not changed

__Move______

- Code: https://github.com/karthik/rdrop2/blob/master/R/drop_file_ops.R
- Change from v1: /1/fileops/move to v2: /2/files/move
- Return values and metadata have changed
- ‘root’ not available in v2, from_path and to_path have not changed

__Delete__

- Code: https://github.com/karthik/rdrop2/blob/master/R/drop_file_ops.R
- Change from v1: /1/fileops/move to v2: /2/files/move
- Return values and metadata have changed
‘root’ not available in v2, from_path and to_path have not changed

__Create folder__

- Code: https://github.com/karthik/rdrop2/blob/master/R/drop_file_ops.R
- Change from v1: /1/fileops/create_folder to v2: /2/files/create_folder
- Return values and metadata have changed
‘root’ not available in v2, ‘path’ has not changed

**Comment by ClaytonJY** [comment]
some notes from outside discussion:
- `drop_exists` should probably use `drop_get_metadata` rather than `drop_dir`
- try to use `uuid` when refactoring tests; the "existential test" in particular hits a race condition almost every time two travis builds are running simultaneously

**Comment by ClaytonJY** [comment]
Closed by #121.
